packages/cli/lib/upgrade.js

```diff
@@ -21,8 +22,12 @@ export async function upgrade (argv) {
       config: 'c'
     }
   })
+  await upgradeApp(args.config)
+  await upgradeSystem()
```

Comment:
Can you wrap this in a try-catch? If it errors it should be silently ignored.

Reply:
Should we ignore them and let the user not see anything?
I think it would be better to `console.error(err.message)` at least...